### ISSUE TITLE
Comment on the pull request

### ISSUE THREAD

**Issue by kevgo** [amannn/action-semantic-pull-request#187]
It would be nice if this action would also add a comment to the pull request directly. Here is an example for how that could look like:

![image](https://user-images.githubusercontent.com/268934/176431582-5e062cfd-c144-4ffd-b556-2eb774b8172e.png)

The message should - of course - be configurable via an input option.

Implementing this is pretty straightforward:

```ts
github.rest.issues.createComment({
  issue_number: context.issue.number,
  owner: context.repo.owner,
  repo: context.repo.repo,
  body: "Please make the title of this Pull Request follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) format 🙏"
})
```

**Comment by amannn** [comment]
Can this be achieved with https://github.com/amannn/action-semantic-pull-request/issues/178?

**Comment by jthegedus** [comment]
As I state above, I think commenting on PRs and Action outputs are different. Can we reopen this or should I create another Issue to specifically track comments on PR feature request?

**Comment by jthegedus** [comment]
>Having a comment repeat what a check already provides doesn't seem useful to me.

I disagree. Seeing that the check failed does not tell you **why** it failed. You have to click through via the "details" link to see the Action execution details. From my experience on `asdf`, no external contributors understand that this check failed or why, no one click throughs, and the maintainers have to resolve semantic-pr issues. Contributors are more likely to respond to comments on their PRs than the checks themselves. The education piece is missing.

**Comment by kevgo** [comment]
At our project, many non-developers use GitHub pull requests to manage things like blog posts or job postings. Most of these users are not comfortable clicking on a broken CI status and sifting through very technical steps and their console output.

I think getting a friendly comment on a pull request that explains in plain English (or French|Spanish|Chinese) what exactly the problem is and links to further documentation is a lot more user friendly for these people than the current behavior.

I think it would be completely okay if this feature is disabled by default. This GitHub Action doesn't even need to provide a default text. All that is needed from my perspective is a string input parameter that when set posts the given text as a comment on the pull request.
